### PR TITLE
Myaccount

### PR DESCRIPTION
**Pull request by hardik-isobar**
My account page and Personal Information Page

### CODE REVIEW

File: src/components/myAccount/myAccountLinks.vue [hunk 1]
```diff
@@ -0,0 +1,84 @@
+<template>
```
Comment (rahimoddin) [inline src/components/myAccount/myAccountLinks.vue:1]: We are following Capital case for file names. Please rename it.

File: src/store/index.js [hunk 2]
```diff
@@ -25,6 +26,7 @@ export default new Vuex.Store({
     cart,
     payment,
     shipping,
-    wishlist
+    wishlist,
```
Comment (rahimoddin) [inline src/store/index.js:29]: Can we remoe the store for wishlist and add it as part of myaccount store? Just a suggestion.

File: src/store/modules/myaccount.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            dd:'15',
+            mm:'09',
+            yy:'1980'
+        },
+        email:'john.smith@gmail.com'
+    }
+}
+
+const getters = {
+    getPersonalInfo(state) {
```
Comment (rahimoddin) [inline src/store/modules/myaccount.js:16]: does this getter needed? we can map the state variable itself?

File: src/components/myAccount/myAccountLinks.vue (hunk 1 above)
Comment (hardik-isobar) [reply src/components/myAccount/myAccountLinks.vue:1]: Done

File: src/store/index.js (hunk 2 above)
Comment (hardik-isobar) [reply src/store/index.js]: Done

File: src/store/modules/myaccount.js (hunk 3 above)
Comment (wasil-sayyed) [reply src/store/modules/myaccount.js:17]: yes we can map state, but getters serve the same purpose which makes code DRY and comes under best practices also.

**Pull request merged**